src/prop/prop_engine.cpp

```diff
@@ -394,25 +394,6 @@ bool PropEngine::isFixed(TNode lit) const
   return false;
 }
 
-void PropEngine::printSatisfyingAssignment(){
```

Comment:
What was the motivation to push this down into the CnfStream? `traceSatisfyingAssignment` is only used in the PropEngine atm

Reply:
After refactoring, this was the only piece of code using the `getTranslationCache` method. It seemed appropriate to remove it from the interface at the time, but I can revert the change if necessary.